src/components/terms-and-conditions.tsx

```diff
@@ -59,8 +59,7 @@ export const TermsAndConditions = ({
         </div>
         <span>.</span>
       </label>
-      <p className="-mt-4 text-xs text-red-500">{fieldState.error?.message}</p>
-
+      <p className="-mt-4 text-xs text-red-500">{fieldState.error?.message}</p>s
```

Comment:
Remove the s at the end of the line. While you're at it, can you make this only render when there's an error?

This should do it
```ts
!!fieldState.error?.message && <p>...</p>
```